src/webportal/src/app/job/job-view/fabric/task-attempt/task-attempt-dialog.jsx

```diff
@@ -1,75 +0,0 @@
-// Copyright (c) Microsoft Corporation
```

Comment:
Why this file is deleted?

Reply:
It is an out-dated file, which designed to open a dialog. And the new detail will open a new page instead of a dialog, so it is deleted